hw/bus/syscfg.yml

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+    BUS_DRIVER_PRESENT:
```

Comment:
This definition got me confused for a while, why value should not be overridden, but I think it would be beneficial to have **newt** generated value derived from package name and path that would serve this purpose.
The **newt** tool could add something like **_HW_BUS_PRESENT**.
What do you think @ccollins476ad @mkiiskila ?

Reply:
I think this is a good idea, few other packages could benefit from such addition (e.g. NimBLE).
perhaps packages could define smth like features it implements and then newt would generate MYNEWT_FEATURE macro similar to MYNEWT_VAL. I'd rather not use package name as a base since we can eventually move some packages so name can change and we'll have problem then :-)